### DISCUSSION TITLE
SR isn't propogated everywhere in the song select screen after editing

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16186] [Q&A] [answered]
After finishing mapping a beatmap from scratch, I exited the editor to find that the SR had not yet been calculated. I assumed that it would take a moment, but after waiting a few minutes, there was no change. However, upon applying the Auto mod, it immediately showed the correct SR. Removing Auto set the SR back to 0.00. After restarting the client, the SR shows correctly everywhere except for the mapset banner(? not sure what this is called).
Version 2021.1218.0
![osu_2021-12-20_18-34-32](https://user-images.githubusercontent.com/74262407/146863810-b44ae171-82f8-41e3-a058-95551c2cd9b5.png)
![osu_2021-12-20_18-34-34](https://user-images.githubusercontent.com/74262407/146863814-b1b1014c-5654-4955-b252-57291734ff49.png)
![osu_2021-12-20_18-56-06](https://user-images.githubusercontent.com/74262407/146863816-d2a46099-772e-46ee-aa31-240238ec9d65.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/12048

this is semi blocked on the database migration to realm.
